package.json

```diff
@@ -38,7 +38,7 @@
     "react": "16.13.1",
     "react-dom": "16.13.1",
     "react-native": "https://github.com/expo/react-native/archive/sdk-42.0.0.tar.gz",
-    "react-native-calendars": "^1.1264.0",
+    "react-native-calendars": "^1.17.0",
```

Comment:
@YajanaRao Apparently this is a very old version of react-native-calendars which was released in 2018. The latest is actually `1.1267.0` which #103 upgraded. Can you test web on that version and get rid of this downgrade?

Reply:
Resolved